exotations/solvers/aico/src/AICOsolver.cpp

```diff
@@ -106,7 +106,6 @@ void AICOsolver::specifyProblem(PlanningProblem_ptr problem)
     MotionSolver::specifyProblem(problem);
     prob_ = std::static_pointer_cast<UnconstrainedTimeIndexedProblem>(problem);
 
-    T = prob_->T;
     initMessages();
```

Comment:
```initMessages()``` uses ```T``` to resize internal vectors. If you call ```Problem.setT()``` and then ```solve``` the solver will crash.

Reply:
Good catch - I've added c272d99 to address this.